### PR TITLE
Implement country subdivisions

### LINKED ISSUE #228: Add ISO_3166-2 to support country sub-divisions
**Problem**

My org needs more localised areas than countries and would like to use this library.

**Solution**
Add [ISO_3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) to this project to include country subdivisions.

### PR DESCRIPTION
**Pull request by PrinsFrank**
Fixes #228

### CODE REVIEW

File: src/Country/Subdivision/CountrySubdivision.php [hunk 1]
```diff
@@ -0,0 +1,5053 @@
...
+    case Azerbaijan_Goygol = 'AZ-GYG';
+    case Azerbaijan_Haciqabul = 'AZ-HAC';
+    case Azerbaijan_Imisli = 'AZ-IMI';
+    case Azerbaijan_Ismayilli = 'AZ-ISM';
+    case Azerbaijan_Klbcr = 'AZ-KAL';
+    case Azerbaijan_Kngrli = 'AZ-KAN';
+    case Azerbaijan_Kurdmir = 'AZ-KUR';
+    case Azerbaijan_Lnkran = 'AZ-LA';
+    case Azerbaijan_Lacin = 'AZ-LAC';
+    case Azerbaijan_Lnkran = 'AZ-LAN';
```
Comment (szepeviktor) [inline src/Country/Subdivision/CountrySubdivision.php:176]: @PrinsFrank What transliteration produces two `Azerbaijan_Lnkran`-s?

Comment (szepeviktor) [reply src/Country/Subdivision/CountrySubdivision.php:176]: image

Comment (szepeviktor) [reply src/Country/Subdivision/CountrySubdivision.php:176]: Don't we have a mechanism to add a suffix to the second one?

Comment (PrinsFrank) [reply src/Country/Subdivision/CountrySubdivision.php]: We do, but that only works where different source scripts result in the same transliteration. The source scripts match in this case. I've decided to prefix with subdivision type to make the keys unique instead.

File: .typos.toml [hunk 2]
```diff
@@ -15,5 +15,30 @@ extend-ignore-re = [
 [default.extend-identifiers]
 "Viet_Nam" = "Viet_Nam"
 "Viet_Nam_Socialist_Republic_of" = "Viet_Nam_Socialist_Republic_of"
+"GOST" = "GOST"
```
Comment (szepeviktor) [inline .typos.toml:18]: @PrinsFrank Biog NO! Please see my PR #235

Comment (PrinsFrank) [reply .typos.toml]: Yeah, that's way better, sorry about that. ;)

**Comment by PrinsFrank** [comment]
@szepeviktor The extend-ignore-re doesn't work as expected. Do you see what's going on?

File: .typos.toml [hunk 3]
```diff
@@ -10,6 +10,7 @@ extend-ignore-re = [
     "\\bself::[0-9A-Z_a-z]+ +=> .+,",
     "\\b(Country|Currency)[:0-9A-Z_a-z]+,",
     "\\b(Country|Currency)[:0-9A-Z_a-z]+ +=> [:0-9A-Z_a-z]+,",
+    "\\b#\\[Name\\(.+\\[LanguageAlpha2::.+",
```
Comment (szepeviktor) [inline .typos.toml:13]:
```suggestion
    "#\\[Name\\(.+\\[LanguageAlpha2::.+",
```

`#` is no word!

Comment (PrinsFrank) [reply .typos.toml:13]: Ah, cool, thanks for the assistance!

**Review by szepeviktor — APPROVED** [review]

**Pull request merged**